ci/test-stable.sh

```diff
@@ -29,7 +29,7 @@ done
 
 # Run integration tests serially
 OLD_RUST_LOG=$RUST_LOG
-for test in tests/*.rs wallet/tests/*.rs drone/tests/*.rs; do
+for test in tests/*.rs wallet/tests/*.rs drone/tests/*.rs ledger-tool/tests/*.rs; do
```

Comment:
`*/tests/*.rs`?

Reply:
Yeah that's more better.   The glob we need is `{,*}tests/*.rs` to match `tests/*.rs` as well

Reply:
Should it maybe be: `{,*/}tests/*.rs`?